src/main/kotlin/com/example/tlitter/controllers/UserController.kt

```diff
@@ -0,0 +1,26 @@
+package com.example.tlitter.controllers
+
+import io.swagger.annotations.ApiOperation
+import io.swagger.annotations.ApiResponse
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping("/api/users")
+class UserController {
+    @PostMapping("login")
+    @ApiOperation(value = "認証を行いJWTを返す")
+    @ApiResponse(code = 404, message = "不正なEmailまたはPasswordです")
+    fun login(@RequestBody loginDTO: LoginDTO): JwtDTO {
+        return JwtDTO("This is JWT!")
+    }
+
+    class LoginDTO(
+        val email: String,
+        val password: String
+    )
+
+    class JwtDTO(val jwt: String)
```

Comment:
[imo]
requestに同じくで(JwtResponseはなんとなく違和感ありますが...)
```suggestion
    class JwtResponse(val jwt: String)
```

Reply:
上に同じく、こちらで対応しました！ 2d01476

名前は `TokenResponse` としています
意図としては、Responseに具体的な名前（JWT）を入れる必要がないかも？といった判断です